### PR TITLE
Doc - Upgrade Refactor - Next Step

### PR DESCRIPTION
**Pull request by sleto-it**
Internal Ref:
- https://github.com/arangodb/planning/issues/2641

Build:
- https://docs.arangodb.com/review/documentation/new-upgrade-refactor/Manual/

### CODE REVIEW

File: Documentation/Books/Manual/Downgrading/README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
 Downgrading
 ===========
 
+A direct, binary, downgrade of ArangoDB is **not** supported. If you have upgraded your ArangoDB package, and then also upgraded your current data directory, it is not supported to downgrade the package and start an older ArangoDB version on a data directory that was upgrade already.
```
Comment (Simran-B) [inline Documentation/Books/Manual/Downgrading/README.md]: Remove extra comma? `A direct, binary downgrade`

What does _binary_ mean in this context?

File: Documentation/Books/Manual/Downgrading/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+1. As extra precaution, take a backup of your current data directory (at filesystem level).
+   If you are using an Active Failover or a Cluster, you will need to backup all the data
+   directories of all the processes involved, from all machines. Make sure you move your
+   data directory backup to a safe place.
+1. Uninstall the ArangoDB package (use appropriate _purge_ option so your current data
+   directory is deleted, if you are using a stand alone instance).
+1. Install the old version of ArangoDB.
+1. Start ArangoDB. If you are using the _Starter_, please make sure you use a new data
+   directory for the _Starter_.
+1. Restore the _arangodump_ backup taken before upgrading.
```
Comment (Simran-B) [inline Documentation/Books/Manual/Downgrading/README.md]: Markdown is supposed to be readable in plaintext as well, but allows this pattern for ordered lists. I would prefer to use sequential numbers in the source however, like

```
1. foo
2. bar
3. baz
```

Comment (sleto-it) [reply Documentation/Books/Manual/Downgrading/README.md]: done.

File: Documentation/Books/Manual/Downgrading/README.md (hunk 1 above)
Comment (sleto-it) [reply Documentation/Books/Manual/Downgrading/README.md]: binary upgrade, or downgrade means in this context: you upgrade or downgrade your binary, then start the server and you do not need to upgrade or restore. this is what we have called "in-place" in https://docs.arangodb.com/devel/Manual/Upgrading/GeneralInfo/  - hope this helps

Comment (graetzer) [reply Documentation/Books/Manual/Downgrading/README.md]: with rocksdb its actually not going to be possible anymore to downgrade to 3.3. 
Startup will just fail when going 3.4 -> 3.3

**Pull request merged**